webpack.config.js

```diff
@@ -36,29 +35,27 @@ const templateFiles = filesToIncludeArr
     output: filename.replace(/\.ejs$/, '.html'),
 }));
 
-const htmlPluginEntries = templateFiles.map((template) => new HTMLWebpackPlugin({
-  inject: true,
-  hash: false,
+const htmlEntries = templateFiles.map((template) => ({
   filename: template.output,
-  template: path.resolve(environment.paths.source, template.input),
-  newVariable: 'test variable',
-  
+  import: path.resolve(environment.paths.source, template.input),
+  // use the `data` option to pass variables into the template
+  data: {
+    // use any object name which will be available in html template
+    pageData: {
+      newVariable: 'test variable',
+    },
+  },
 }));
 
-
 module.exports = {
-  entry: {
-    app: path.resolve(environment.paths.source, 'js', 'app.js'),
-  },
   output: {
-    filename: 'js/[name].js',
```

Comment:
define JS output file in the `js.filename` plugin option to keep all relevant options in one place